sources/logs.datatables.php

```diff
@@ -123,57 +118,61 @@
     $searchValue = (string) $params['sSearch'];
 }
 
+// Ordering
+$order = $params['order'][0] ?? [];
+if (isset($order['dir'], $order['column']) 
+    && in_array($order['dir'], $aSortTypes, true)
+    && (int) $order['column'] !== 0
+    && $order['column'] !== 'asc'
+) {
+    $orderDirection = $order['dir'];
+} else {
+    $orderDirection = 'DESC';
+}
```

Comment:
I think it would be clearer do something like:
```php
$aSortTypes = ['ASC', 'DESC'];

$order = strtoupper($params['order'][0]['dir'] ?? null);
$orderDirection = in_array($order, $aSortTypes, true) 
    ? `$orderDirection` : 'DESC';
```
-> Case insensitive.
-> $orderDirection is initialized before opening  brace.

This seems to be useless here:
```php
&& (int) $order['column'] !== 0
    && $order['column'] !== 'asc'
```

Reply:
Yep you are right